packages/desktop/src/desktop_context.rs

```diff
@@ -185,5 +192,16 @@ pub(super) fn handler(
         SetDecorations(state) => window.set_decorations(state),
 
         DevTool => webview.devtool(),
+
+        Eval(code) => webview
+            .evaluate_script(code.as_str())
+            .expect("eval shouldn't panic"),
     }
 }
+
+/// Get a closure that executes any JavaScript in the WebView context.
+pub fn use_eval<S: std::string::ToString>(cx: &ScopeState) -> impl Fn(S) + '_ {
+    let desktop = use_window(&cx);
+
+    move |script| desktop.eval(script)
+}
```

Comment:
We haven't really been returning `impl Fn` from any of ours hooks - mostly because they can't be shared between two handlers. I think that makes sense here too - eval might need to be used from two handlers. I would suggest - to be more inline with how we return other functions, to return `&dyn Fn`. To be fancier, we could return `&Rc<dyn Fn>` for use in async contexts.